javascript/src/main/pages/RoomSlots/RoomSlots.js

```diff
@@ -36,7 +39,7 @@ const RoomSlots = () => {
         <Button className="mb-3" onClick={()=>history.push("/roomslots/new")} >New Room Slot</Button>
       )}
       {(roomSlotList) && (
-        <RoomSlotTable roomSlots={roomSlotList} />
+        <RoomSlotTable roomSlots={roomSlotList} admin={true} deleteRoomSlot={deleteRoomSlot}/>
```

Comment:
```suggestion
        <RoomSlotTable roomSlots={roomSlotList} admin={isAdmin} deleteRoomSlot={deleteRoomSlot}/>
```
We should pass the `isAdmin` variable evaluated above into the table.